recorder.js

```diff
@@ -33,6 +36,27 @@ function parse_game_stats(){
             }
         });
     });
+    if (team_game) {
+        let team_one_score = document.querySelector('[data-hook=team1-score]').textContent;
+        let team_one_name = teams[0].reduce(
+            function(prev, curr){return prev + ' ' + curr},
+            'Team: '
+        );
+        let team_two_name = teams[1].reduce(
+            function(prev, curr){return prev + ' ' + curr},
+            'Team: '
+        );
+        let team_two_score = document.querySelector('[data-hook=team2-score]').textContent;
+        if (team_one_score > team_two_score){
```

Comment:
these 3 ifs are mutually exclusive, so they could go in a if-else if-else statement.

Reply:
I cut this down pretty far but can't help but feel there's still some improvement to the control flow to be made- but the tie edge case, where I will want to have both teams available and formatted, makes this less elegant.